### ISSUE TITLE
Dietary Tags?

### ISSUE THREAD

**Issue by izrunas** [nazgul26/PHPRecipebook#62]
In our application of this great software, we are using it to organize and manage breakfast recipes for use in our bed and breakfast inn.  We are often asked to respect various dietary requirements and it would be useful if these was a way to add another section of sorting options for the "Recipe Box" where a recipe can be selected via dietary restrictions.

I would envision this as working by adding the option to add checkboxes on the ingredients to allow for selection of things like "Gluten", "Vegan", "Pork", "Dairy/Lactose", "Meat" and a variety of other "things this ingredient contains"

Then, for the recipe box, there should be user-defined categories where a recipe would be included/excluded based on the various checkboxes for the ingredients in the recipes.  For example, for a fruit gaspacho, it should be included in a category like "L-O-Vegetarian" but should excluded from "Vegan" because the recipe contains honey (which is not vegan-compliant) 

I hope this feature request is clear, but I am happy to clarify more and/or help with the development...

**Comment by nazgul26** [comment]
I think I understand.  The most common example I remember of software doing this would be in Gmail.  You can introduce a 'Label' to a message on the fly.  This 'Label' (aka Tag) is then available for future use and searching later.   Right?

Todd

**Comment by izrunas** [comment]
Yes, essentially just the ability to add "tags" to recipes, then be able to refine the list "by course", etc., by said tags.

**Comment by nazgul26** [comment]
Took a while but I finally added this feature.  It is in release 6.8 - https://github.com/nazgul26/PHPRecipebook/releases/tag/6.8

<img width="995" height="668" alt="Image" src="https://github.com/user-attachments/assets/cbbd90a1-03c7-4b3e-add8-dedd60d40dd2" />
